branchpro/apps/_simulation.py

```diff
@@ -0,0 +1,121 @@
+#
+# SimulationApp
+#
+# This file is part of BRANCHPRO
+# (https://github.com/SABS-R3-Epidemiology/branchpro.git) which is released
+# under the BSD 3-clause license. See accompanying LICENSE.md for copyright
+# notice and full license details.
+#
+
+import pandas as pd
+import dash
+import dash_bootstrap_components as dbc
+
+import dash_core_components as dcc
+import dash_html_components as html
+
+import branchpro as bp
+
+
+class IncidenceNumberSimulationApp:
+    """IncidenceNumberSimulationApp Class:
+    Class for the dash app with figure and sliders.
```

Comment:
As it stands, this app is quite specific to branching processes, since it assumes a branching process model and adds those sliders specifically. I would mention that in the docstring